Descent3/pilot.cpp

```diff
@@ -2816,20 +2815,20 @@ void ShipSelectDeleteLogo(newuiListBox *lb) {
   }
 
   // Get the filename
-  std::filesystem::path p = Custom_images[selected_index - 1];
+  std::filesystem::path custom_filename = Custom_images[selected_index - 1];
 
   // delete custom_filename, we don't want it....
   char buffer[512];
   snprintf(buffer, sizeof(buffer), TXT_PLTOKDEL, custom_logoname);
   if (DoMessageBox(TXT_PLTDELCONF, buffer, MSGBOX_YESNO, UICOL_WINDOW_TITLE, UICOL_TEXT_NORMAL)) {
-    LOG_INFO.printf("Deleting pilot logo %s (%s)", custom_logoname, custom_filename);
+    LOG_INFO.printf("Deleting pilot logo %s (%s)", custom_logoname, custom_filename.c_str());
```

Comment:
Please use `custom_filename.u8string().c_str()` in all log messages, otherwise Windows `wchar_t` based `std::fs::path`'s will not be displayed correctly.

Reply:
Thanks for the feedback. I've added `u8string()` in the logging calls.

Reply:
Thanks for for your fix.